macports.css

```diff
@@ -148,6 +148,7 @@ a#download:hover {
 
 #side-navigation dt {
```

Comment:
Since you removed the "Shortcuts" heading there no longer is a `<dl>`, `<dt>` or `<dd>` element, so we can probably drop those rules completely.

Reply:
Done